README.md

```diff
@@ -109,6 +109,42 @@ tfrecorder create-tfrecords \
     --output_dir=gs://my/bucket
 ```
 
+#### From image directory
+
+```python
+import tfrecorder
+
+tfrecorder.create_tfrecords(
+    input_data='/path/to/image_dir',
```

Comment:
This seems like it might be confusing to the user.  Do you think perhaps we should create tfrecorder.create_tfrecords_from_directory() or some variation of that.  I'm nervous input_data=''  is going to be confusing since that can be either a CSV or a directory structure.

Reply:
I see your point, although we do have support for CSV and Pandas DataFrames already with `input_data` (the docstring for that arg also already mentions image directory). The TFRecord creation steps are similar for any of the supported sources as they ultimately end up as a Pandas DataFrame before being passed to the pipeline. We could probably talk about having different `create` functions for each input data type, and then use `build_pipeline` as the common path.

For now, I renamed `input_data` to `source` in case that helps to generalize the arg.